content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md

```diff
@@ -0,0 +1,48 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Rode este comando em qualquer máquina que você deseje adicionar a um cluster
+existente
+
+### Sinopse
+
+
+
+Ao associar um novo nó a um cluster inicializado com kubeadm, temos que
+estabelecer confiança bidirecional. Este processo é dividido entre a descoberta
+(em que o nó estabelece confiança na camada de gerenciamento do Kubernetes) e a
+inicialização TLS (em que a camada de gerenciamento do Kubernetes estabelece
+confiança no nó).
```

Comment:
The same as before "estabelece a confiança no nó".

Reply:
Updated!